en/docs/guides/user-mgt/associating-user-accounts.md

```diff
@@ -0,0 +1,34 @@
+# Associating User Accounts
+
+WSO2 Identity Server (WSO2 IS) allows you to associate multiple accounts
+you may have, and switch between accounts once you associate accounts.
+WSO2 IS also allows you to connect your federated user credentials with
+your WSO2 Identity Server account. This topic provides instructions on
+how to associate all your user accounts to the account with which you
+have logged on.
+         
+Follow one of the two approaches below to associate all your user
+accounts:
+
+## Using the AdminService
+
+The first approach is to use the `         AdminService        ` . You
+can access this admin service using the following URL: '
+`         https://<HOST_NAME>:9443/services/UserAccountAssociationService?wsdl        `
+'. If you are new to admin services, see [Calling Admin
+Services](insert-link).
+
+The following actions can be performed using above admin service.
+
+-   Create a new user account association
+-   Delete an existing user account association
+-   Get all associated user accounts of the logged in user
+-   Switch between associated user accounts without re-authenticate with
+    the system
+
+## Using the user portal
+
+The WSO2 Identity Server [user portal](../../guides/user-portal) can be used to associate the
+accounts. You can associate a local user account or a federated user
+account. See [Linked Accounts](../../guides/user-portal/#linked-accounts) under
+User Portal for details.
```

Comment:
shall we add a point for the association REST API and add a link to it from this page?